bin/check_samplesheet.py

```diff
@@ -102,13 +102,13 @@ def check_samplesheet(args):
                 + args.input.name
                 + ". Valid types are 'taxa', 'bins' and 'assembly'."
             )
-        if type == "taxa" and not fname.lower().endswith((".txt", ".tsv")):
+        if type == "taxa" and not fname.lower().endswith((".txt", ".csv")):
```

Comment:
Shall we keep the txt input format for plain taxon_id lists?

Reply:
I tend not to, because it would not be tested that only for the one column case the txt format is valid